### DISCUSSION TITLE
Register V in Wikipedia's 'list of programming languages' entry

### DISCUSSION THREAD

**Discussion by MohammadMD1383** [vlang/v#18591] [Ideas]
https://en.wikipedia.org/wiki/List_of_programming_languages

![image](https://user-images.githubusercontent.com/69088224/189892848-4aa87562-f805-40da-977a-e4de409a1892.png)
_missing V here ..._

**Comment by depperm** [comment]
Needs a `V (programming language)` page first, which has happened in the past, but it's been deleted because of lack of sources

**Reply by Wajinn** [reply]
Go check now.  That it was deleted depends on who is supporting and making edits.  There are those who don't want to see it and tear it out.  People have to support what they want to exist.  It's there for everybody to enjoy or contribute.

**Reply by medvednikov** [reply]
Seems to work now?

https://en.wikipedia.org/wiki/V_(programming_language)
